components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -232,6 +233,20 @@ - (void)dismiss:(BOOL)accepted {
                            }];
 }
 
+- (void)applyMDCFeatureHighlightViewDefaults {
+  _outerHighlightColor = [self MDCFeatureHighlightDefaultOuterHighlightColor];
+  _innerHighlightColor = [self MDCFeatureHighlightDefaultInnerHighlightColor];
+}
+
+- (UIColor *)MDCFeatureHighlightDefaultOuterHighlightColor {
+  return [[UIColor blueColor] colorWithAlphaComponent:kMDCFeatureHighlightOuterHighlightAlpha];
```

Comment:
How do you expect these methods to be used? If they are simply to initialize the ivars, please consider removing the method indirection altogether and simply assign the ivar in the init method.